src/modules/login/components/MessageBox.js

```diff
@@ -26,14 +25,19 @@ class MessageBox extends Component {
     }
 
     render() {
-        return (
-            <div>
-                {this.checkError() != "" &&
-                    <div className={styles.messageBox}>
-                        {this.checkError()}
-                    </div>}
-            </div>
-        );
+        if (this.checkError() != "") {
```

Comment:
I would rather update state in the error function and check if state has an error message or not, something like: 


``` 
{this.state.errors &&
  <div>{this.state.errors}</div>
```